lib/index.js

```diff
@@ -88,7 +97,7 @@ module.exports = function (crypto) {
   output.decrypt = function (object, password, config) {
```

Comment:
We can use arrow functions and default parameters.

Reply:
Done that.